build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Build docker script
+set -e
+
+#Login into dockerhub
+# Please export dockerhub user and password
+#docker login -u=${DOCKERHUB_USERNAME} -p=${DOCKERHUB_PASSWORD}
+
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
```

Comment:
Prefer using long flags consistently.

Reply:
```suggestion
```
Actually, remove this line altogether. It looks like it was left in accidentally as it doesn't actually do anything but run an unrelated image.

Reply:
Ah, does `docker buildx create --use` implicitly use the previous image used in `docker run`? I'm not familiar with it.